scrapi/settings/local-dist.py

```diff
@@ -10,8 +10,8 @@
 
 STORE_HTTP_TRANSACTIONS = False
 
-NORMALIZED_PROCESSING = ['storage']
-RAW_PROCESSING = ['storage']
+NORMALIZED_PROCESSING = ['storage', 'cassandra']
```

Comment:
yeah, I mostly stuck it here because we didn't have a travis settings file, which we probably should.